### PR TITLE
add target-mssql

### PR DESCRIPTION
**Pull request by pnadolny13**
Adding target-mssql so its available but labeled as development. Related to @radbrt's post in slack https://meltano.slack.com/archives/C01RKUVUG4S/p1668520638602389?thread_ts=1668199751.268159&cid=C01RKUVUG4S.

@radbrt is it still considered in development or is this something that a new user could use with success? If not, how much do you think it will take to get it to a stable place?

cc @edgarrmondragon @aaronsteers @kgpayne since I know you all were assisting with target SDK quirks.

### CODE REVIEW

**Comment by pnadolny13** [comment]
@edgarrmondragon the plugin definition test is failing, do you know if `schema-flattening` is something we should add to the enum of valid capabilities? It was listed using `--about`.

**Comment by cjohnhanson** [comment]
@pnadolny13 I'll defer to @edgarrmondragon if he has a good reason why it shouldn't be added, butI think `schema-flattening` should probably be added to that enum. It's [in the plugin capabilities enum in the sdk](https://github.com/meltano/sdk/blob/ca27fd5aba27d248dc99756885033170cd6d1fb1/singer_sdk/helpers/capabilities.py#L178).

**Review by cjohnhanson — APPROVED** [review]
Approving, conditional on `schema-flattening` being added to capabilities in the json schema for loaders.

File: _data/default_variants.yml
Comment (tayloramurphy) [inline _data/default_variants.yml:430]: Can we make this alphabetized?

File: _data/meltano/loaders/target-mssql/storebrand.yml
Comment (tayloramurphy) [inline _data/meltano/loaders/target-mssql/storebrand.yml:4]: I found https://images.app.goo.gl/szsXvxMyuBb29ErZ9 which seems to be the logo w/o the wordmark - can we use that?

File: _data/meltano/loaders/target-mssql/storebrand.yml
Comment (tayloramurphy) [inline _data/meltano/loaders/target-mssql/storebrand.yml:21]:
```suggestion
  label: SQLAlchemy Url
```

**Review by tayloramurphy — CHANGES_REQUESTED** [review]

**Comment by radbrt** [comment]
Wow, this happened more quickly than I anticipated. I haven't polished the meltano.yml file or anything, and I just now created a pypi package (I understand that is not strictly necessary though).

Regarding @pnadolny13's questions above: I am still developing the target, but I believe the limitations now listed in the readme are few enough that some people could find it useful as it is now. Marking it as under development is key though, while I don't think there will be any breaking changes, I can't promise it.

File: _data/meltano/loaders/target-mssql/storebrand.yml
Comment (edgarrmondragon) [inline _data/meltano/loaders/target-mssql/storebrand.yml:8]: 👍

**Review by edgarrmondragon — APPROVED** [review]

File: _data/meltano/loaders/target-mssql/storebrand.yml
Comment (edgarrmondragon) [reply _data/meltano/loaders/target-mssql/storebrand.yml:8]: @pnadolny13 I agree that this should probably be added to the JSON schema enum

**Comment by pnadolny13** [comment]
@radbrt yeah I wanted to get it up before I forgot! This at least shows others that something exists so they dont start building another variant. Hopefully you'll start getting some good early feedback and get bugs raised faster. The development maintenance status should make it clear to users that you're still working.

**Review by tayloramurphy — APPROVED** [review]

**Pull request merged**
